official/benchmark/models/resnet_imagenet_main.py

```diff
@@ -148,8 +182,8 @@ def run(flags_obj):
       boundaries=list(p[1] for p in common.LR_SCHEDULE[1:]),
       multipliers=list(p[0] for p in common.LR_SCHEDULE),
       compute_lr_on_cpu=True)
-  steps_per_epoch = (
```

Comment:
If this is a change due to some linter, please check it's the same as the one that Official Models uses (if they mention it).

Reply:
I see similar indenting changes everywhere